src/js/module.ts

```diff
@@ -219,3 +226,49 @@ export function initializeFileSystem(Module: Module, config: ConfigType) {
   mountLocalDirectories(Module, config._node_mounts);
   Module.preRun.push(() => initializeNativeFS(Module));
 }
+
+export function preloadWasm(Module: Module, indexURL: string) {
+  if (SOURCEMAP) {
+    // According to the docs:
+    //
+    // "Sanitizers or source map is currently not supported if overriding
+    // WebAssembly instantiation with Module.instantiateWasm."
+    // https://emscripten.org/docs/api_reference/module.html?highlight=instantiatewasm#Module.instantiateWasm
+    return;
+  }
+  const { binary, response } = getBinaryResponse(
+    indexURL! + "pyodide.asm.wasm",
```

Comment:
nitpick: is `!` needed? `indexURL` is always a string.

Reply:
Looks like you're right. It is used like this in a couple of places.